### ISSUE TITLE
add allcontributors integration to add them all to README.md

### ISSUE THREAD

**Issue by yarikoptic** [incf-nidash/PyNIDM#368]
instigated by https://github.com/incf-nidash/PyNIDM/pull/363 

We might even want to adopt "our" https://github.com/con/tributors which we ATM use within https://github.com/datalad/datalad/blob/maint/.github/workflows/update-contributors.yml to fully automate.

It is somewhat blocked by https://github.com/incf-nidash/PyNIDM/issues/358#issuecomment-1535434318 in that we might better first strip README.md from all documentation which should go into `docs/` so might better come later but could be done even now.

**Comment by dbkeator** [comment]
I agree, would be great to include all contributors and automate this
